api/templates/history-employment.xml

```diff
@@ -104,9 +104,9 @@
         {{- if eq (branch $Dismissal.Has) "Yes"}}
         <Dismissal ID="{{inc $lindex}}">
           <Date Type="{{dateEstimated $Dismissal.Date}}">{{monthYear $Dismissal.Date}}</Date>
-          <Reason>{{textarea $Item.ReasonLeft.props.ReasonDescription}}</Reason>
```

Comment:
Is `ReasonLeft.props.ReasonDescription` no longer used anywhere?

Reply:
Yeah, just in a different part of this section – history-employment.xml" line 160.